client/scripts/controllers/chain/community/main.ts

```diff
@@ -22,6 +22,9 @@ class Community extends ICommunityAdapter<Coin, OffchainAccount> {
       community: this.id,
       jwt: this.app.user.jwt,
     });
+    if (this.meta.id !== m.route.get().split('/')[1]) {
```

Comment:
this seems fragile, is there a way to use m.route.get() to get the routing parameter instead?

Reply:
Good call; I'll look into it!